README.md

```diff
@@ -5,3 +5,60 @@ We will slowly add all our processes in this repo. This is how we roll at MeiliS
 ## Our Mission
 
 We help developers be successful by nurturing a healthy and open-source community around MeiliSearch. This means listening to, inspiring, educating, and supporting our community however we can.
+
+## Activities
+Here is an exhaustive list of developer relations activity categories. We are taking activities that could be managed by a marketing or social media team for now.
+
+### Blog Management
+- Writing blog posts for MeiliSearch or as a guest elsewhere
+- Managing external writers, whether volunteers or paid freelancers
+
+### Code Contributions Managemeny
+- Helping contributors and reviewing code-related PRs
+
+### Documentation Management
+- Helping contributors and reviewing documentation-related PRs
+- Writing documentation
+- Resolving documentation-related issues and maintaining the repository
+
+### Learning Resource Creation
+- Creating demo applications using MeiliSearch
+- Creating helpful code snippets
+- Creating example datasets
+
+### Technical Support / Customer Service:
+- Discourse Support
+- Slack Support
+
+### Hackathon
+- Running hackathon contests and mentoring participants
+
+### Newsletter
+- Writing and sending the newsletter
+
+### Podcast
+- Participating in external audio podcasts or running the official MeiliSearch podcast, if we create one
+
+### Public Speaking
+- Speaking at conferences, including creating slides & demos
+- Organizing and speaking at meetups, including creating slides & demos
+
+### Slack Community Management
+- Ensuring a healthy and active Slack community
+
+### Video Production
+- Participating in or creating our own vidcasts, live streams, and other video-based content (such as tutorials)Social Media Management
+- Posting, commenting, and replying on social media accounts
+
+### Video Production
+- Participating in or creating our own vidcasts, live streams, and other video-based content (such as tutorials)Sponsorship Management
+- Managing sponsorship requests & supporting events we choose to sponsor
+
+### Tech Media Spokesperson
+- Speaking to tech media on behalf of the company
+
+### Video Production
```

Comment:
I'm confused about the 3 Video Production...

Reply:
Mistake, thanks. You can review the new version please.